connection.php

```diff
@@ -0,0 +1,725 @@
+<?php
+
+/**
+* @author: mfsi-krishnadev
+* @class : database connection and operation
+* 
+*/
+class db_connection 
+{
+	//declaration of variables
```

Comment:
Spaces are not constant for aligned.
it 2, 4, 8 at different places. make sure to give only 4 space for alignment.
there is setting in the editor as well to set spaces as 4 space and if tab is pressed then that will be considered as 4 spaced.  so set that configuration if you haven't. If you do not know how to do that then we can meet and check.